src/pages/clients.tsx

```diff
@@ -1,11 +1,81 @@
-import React from 'react';
 import Layout from '@theme/Layout';
+import clsx from 'clsx';
+import React, { useState } from 'react';
+
+import ClientDetails from '../components/ClientDetails';
+import { Clients, DeviceType, Platform } from '../data/clients';
+
+type ClientFilter = {
+  recommended: boolean,
+  deviceType?: DeviceType,
+  platform?: Platform
+}
 
 export default function Plugins() {
+  const [filter, setFilter] = useState<ClientFilter>({
+    recommended: true
+  });
+
   return (
-    <Layout>
-      <h1>Clients</h1>
-      <p>Placeholder page</p>
+    <Layout title='Clients'>
+      <h1 className='text--center'>Clients</h1>
+
+      <main className='margin-vert--lg'>
+        <section className='container'>
+          <ul className='pills'>
+            <li
```

Comment:
Wouldn't it be possible to use native <button> elements here?

Reply:
Native elements like `<button>`? or create a `<Pill>` component in react?

I had attempted to use a button for these instead of using the role + key event handler but the css framework seems to not support them unfortunately. :confused:

Reply:
Native element as in the button element. that would help with the enter key

But if the CSS doesn't work with it I guess this will do for now